packages/repository/src/define-repository-class.ts

```diff
@@ -125,75 +141,30 @@ export function defineCrudRepositoryClass<
 export function defineEntityCrudRepositoryClass<
```

Comment:
I'd like to discuss the name of this function.

The actual base repository class is called `DefaultCrudRepository` (there is no `Entity` in the name). From what I remember, it was an intentional decision - we don't provide non-entity Crud repository implementation, the only Crud repository we have requires an Entity.

I think we should rename the class factory function to `defineCrudRepositoryClass` for consistence.

Note that `@loopback/rest-crud` is already using the name `defineCrudRepositoryClass`.

Thoughts?

Reply:
That's fine with me.